src/Console/Command/Process/Pool/Server/Start.php

```diff
@@ -40,7 +40,9 @@ public function _execute(): CommandAbstract
         pcntl_exec(__DIR__ . '/../../../../../../bin/kojo', $arguments);
         $this->_getOutput()->writeln('An error occurred trying to start the process pool server.');
 
-        return $this;
+        // currently there is no graceful shutdown flow for kojo, it works until terminated
+        // if, in the future, we want such a flow, and want to expose its success via exit code, we will need to make changes here
+        return 255;
```

Comment:
[ProcessAbstract](https://github.com/neighborhoods/kojo/blob/0bb6bd0b159ad6eb55fb714348491d953f8ce54d/src/ProcessAbstract.php#L24) uses a pattern where it sets a `protected $_exitCode = 0;`. Then of things become unhappy, it updats it via:

```php
protected function _setOrReplaceExitCode(int $exitCode): ProcessInterface
    {
        $this->_exitCode = $exitCode;

        return $this;
    }
```

Then when it's time to exit, it used a function that logs, does some other things, and then calls that exit code:
```
public function exit(): void
    {
        $this->unregisterShutdownMethod();
        $this->_getProcessPool()->terminateChildProcesses();
        $this->_getLogger()->debug("Exiting.");
        exit($this->_exitCode);
    }
```


Can this pattern be replicated in CommandAbstract to replace the comment block in this PR?

Reply:
Actually, I'm dumb, this is PHP's wrapper for the `exec()` syscall`. As the error message above says, this code is only executed if we failed to re-run `bin/kojo`

I'll remove my comment because it's not relevant, and I'll replace it with a clarifying comment about why reaching this code is a failure state